.github/workflows/fetch_repos.yml

```diff
@@ -13,6 +13,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # pin@v3.5.3
+      - uses: conda-incubator/setup-miniconda@v3
+        with:
+          auto-update-conda: true
+          python-version: "3.10"
 
       - name: Add conda to system path
```

Comment:
@Abellegese we don't need to have this step for adding conda to system path if we are using that action.